content/en/blog/_posts/2023-08-15-kubernetes-1.28-blog.md

```diff
@@ -136,22 +136,17 @@ CDI provides a standardized way of injecting complex devices into a container (i
 ## API awareness of sidecar containers (alpha) {#sidecar-init-containers}
 
 Kubernetes 1.28 introduces an alpha `restartPolicy` field for [init containers](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/pods/init-containers.md),
-and uses that to indicate when an init container is also a _sidecar container_. The will start init containers with `restartPolicy: Always` in the order they are defined, along with other init containers. Instead of waiting for that sidecar container to complete before starting the main container(s) for the Pod, the kubelet only waits for
+and uses that to indicate when an init container is also a _sidecar container_. The kubelet will start init containers with `restartPolicy: Always` in the order they are defined, along with other init containers. Instead of waiting for that sidecar container to complete before starting the main container(s) for the Pod, the kubelet only waits for
 the sidecar init container to have started.
 
-The condition for startup completion will be that the startup probe succeeded (or if no startup probe is defined) and postStart handler is completed. This condition is represented with the field Started of ContainerStatus type. See the section "Pod startup completed condition" for considerations on picking this signal.
+The condition for startup completion will be that the startup probe succeeded (or immediately if no startup probe is defined). This condition is represented with the field Started of ContainerStatus type.
```

Comment:
I'm trying to make this as idiomatic as possible because it still reads confusing to me. Please let me know if it is technically accurate.

```suggestion
The kubelet will consider the startup for the sidecar container as being completed if the startup probe succeeds. This condition is represented with the field Started of ContainerStatus type. If you do not define a startup probe, the kubelet will consider the container startup to be completed immediately. 
```

Reply:
@gjkim42 added back the postStart handler
@divya-mohan0209 tried to use your reformulation

Please check if it works this way :)

Reply:
Thanks, it is consistent with the current behavior.